### PR TITLE
#1246: Calendar opens on medication outcome select <Resolved>

### PR DESCRIPTION
**Pull request by besong23**
Removed focus from End Date field on the Add issues form so that calendar does not pop up when outcome <Resolved> is selected.

### CODE REVIEW

File: interface/patient_file/summary/add_edit_issue.php [hunk 1]
```diff
@@ -552,7 +552,6 @@ function outcomeClicked(cb) {
    var today = new Date();
    f.form_end.value = '' + (today.getYear() + 1900) + '-' +
     ("0" + (today.getMonth() + 1)).slice(-2) + '-' + ("0" + today.getDate()).slice(-2);
-   f.form_end.focus();
```
Comment (muarachmann) [inline interface/patient_file/summary/add_edit_issue.php]: 👍 this does the trick. Please always describe your PR what it does a suitable name would be **Fix: issue title** no need placing the issue number cause github references issues when they are related or mentioned in PRs.
Approving this

**Comment by muarachmann** [comment]
@aethelwulffe , @teryhill this is ok have a look

File: interface/patient_file/summary/add_edit_issue.php (hunk 1 above)
Comment (besong23) [reply interface/patient_file/summary/add_edit_issue.php]: Okay, will keep that in mind

Comment (aethelwulffe) [reply interface/patient_file/summary/add_edit_issue.php]: Nice description like "Changed focus thingie on my first step to becoming a major daily contributor to LibreHealthEHR" or something like that is usually good to have.

Comment (besong23) [reply interface/patient_file/summary/add_edit_issue.php]: I have added a description to my pull request. Will remember to always do so.

**Pull request merged**